configure.ac

```diff
@@ -709,6 +709,9 @@ then
 
     # Enable deterministic ECC signing API with variant
     AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_ECDSA_DETERMINISTIC_K_VARIANT"
+
+    # Store issuer name components when parsing certificates.
+    AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_HAVE_ISSUER_NAMES"
```

Comment:
Before my most recent push, this line was

```
AM_CFLAGS="-DWOLFSSL_HAVE_ISSUER_NAMES"
```

which wiped out all the CFLAGS set before, including `WOLFSSL_SMALL_STACK_STATIC`. This caused the stack size check to fail in test.c for that PRB where we check the stack frame size at compile time.